test/widgets/action_sheet_test.dart

```diff
@@ -113,37 +113,37 @@ void main() {
     final someMessage = eg.streamMessage(
       stream: someChannel, topic: someTopic, sender: eg.otherUser);
 
-    group('showTopicActionSheet', () {
-      Future<void> prepare({
-        ZulipStream? channel,
-        String topic = someTopic,
-        bool isChannelSubscribed = true,
-        bool? isChannelMuted,
-        UserTopicVisibilityPolicy? visibilityPolicy,
-        UnreadMessagesSnapshot? unreadMsgs,
-        int? zulipFeatureLevel,
-      }) async {
-        final effectiveChannel = channel ?? someChannel;
-        assert(isChannelSubscribed || isChannelMuted == null);
-
-        addTearDown(testBinding.reset);
-
-        final account = eg.selfAccount.copyWith(zulipFeatureLevel: zulipFeatureLevel);
-        await testBinding.globalStore.add(account, eg.initialSnapshot(
-          realmUsers: [eg.selfUser, eg.otherUser],
-          streams: [effectiveChannel],
-          subscriptions: isChannelSubscribed
-            ? [eg.subscription(effectiveChannel, isMuted: isChannelMuted ?? false)]
-            : null,
-          userTopics: visibilityPolicy != null
-            ? [eg.userTopicItem(effectiveChannel, topic, visibilityPolicy)]
-            : null,
-          unreadMsgs: unreadMsgs,
-          zulipFeatureLevel: zulipFeatureLevel));
-        store = await testBinding.globalStore.perAccount(eg.selfAccount.id);
-        connection = store.connection as FakeApiConnection;
-      }
+    Future<void> prepare({
+      ZulipStream? channel,
+      String topic = someTopic,
+      bool isChannelSubscribed = true,
+      bool? isChannelMuted,
+      UserTopicVisibilityPolicy? visibilityPolicy,
+      UnreadMessagesSnapshot? unreadMsgs,
+      int? zulipFeatureLevel,
```

Comment:
I agree that these will potentially become useful as we add more. Maybe at some point it will be helpful to extract some of this to `test/example_data.dart` as a helper to set up global store, along with some invariant checks. Out of scope for this PR, though.